deploy/role.yaml

```diff
@@ -47,8 +47,10 @@ rules:
   verbs:
   - '*'
 - apiGroups:
-  - ""
+  - "*"
```

Comment:
Thanks! Good spot :-) That caused issues in my testing and I forgot to look at it, so took the freedom to copy it. Including the `verbs`.

Reply:
This almost killed me :D